Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -184,12 +184,16 @@ open class ChatMessageListVC: _ViewController,
         listView.delegate = self
         listView.dataSource = self
         listView.reloadData()
+        DispatchQueue.main.async {
```

Comment:
```suggestion
        DispatchQueue.main.async { [weak self] in
```
nit: No reason to hold a strong reference here

Reply:
Clearly there as no retain cycles here, but yes if self is already deallocated, it does not make sense for the closure to still run, so I will add the weak self 👍